cmd/telepilotd/main.go

```diff
@@ -60,11 +61,14 @@ func main() {
 	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
 	defer cancel()
 
+	doneCh := make(chan struct{})
```

Comment:
I think we are missing a `close(doneCh)`.

Reply:
Indeed, missed that. Added in the next PR https://github.com/creack/telepilot/pull/4/files#diff-6874be0d27e65e29ad9c2b6d289609f047856bd871b828395c88f4e52ec5bf34R36